assets/src/dashboard/app/views/myStories/header/index.js

```diff
@@ -66,6 +73,22 @@ function Header({
 
   const { capabilities: { canReadPrivatePosts } = {} } = useConfig();
 
+  const searchOptions = useMemo(() => {
+    // todo add different option sets, value and label won't always be the same
+    return stories.reduce((acc, story) => {
+      if (!story.title || story.title.trim().length <= 0) {
+        return acc;
+      }
+      return [
+        ...acc,
+        {
+          label: story.title,
+          value: story.title,
+        },
+      ];
+    }, []);
+  }, [stories]);
+
```

Comment:
memoizing options here instead of in the old shared `typeahead` component. No changes in how this is getting reduced from before.